src/Context/RestContext.php

```diff
@@ -136,10 +136,11 @@ public function theHeaderShouldNotBeEqualTo($name, $value) {
      *
      * @Then the header :name should contain :value
      */
-    public function theHeaderShouldBeContains($name, $value)
+    public function theHeaderShouldContain($name, $value)
```

Comment:
This break the backwards compatibility. Keep the old one and trigger a deprecated error.

Like this: https://github.com/Behatch/contexts/blob/fc41c076f036fdb445d5e77d6d582951c0c64825/src/Context/SystemContext.php#L135-L142

Reply:
@sanpii fixed